### DISCUSSION TITLE
Custom page

### DISCUSSION THREAD

**Discussion by elainecyx** [filamentphp/filament#13421] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.88

### How can we help you?

I have a custom page which menu can link with it, there a table, filters, search, actions(for enable or disable table row status) in custom page. I'm not sure if this is the best practice to add a table to a Livewire component and then render Livewire component on the custom page? I am a filament and livewire beginner. Thanks in advance.

**Comment by leandrocfe** [comment, marked as answer]
If you don't want to use a resource, yes
https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component#adding-the-table
